Week2/homework-midy/favouriteHobby/favoriteHobbies.js

```diff
@@ -0,0 +1,18 @@
+const myHobbies = [
+  'Meditation',
+  'Reading',
+  'Programming',
+  'Hanging out with friends',
+  'Going to the gym',
+  'Programming again',
+];
+
+const unorderedList = document.createElement('ul');
+
+myHobbies.forEach(i => {
```

Comment:
You can use a more descriptive name instead of `i`. eg `hobby`

Reply:
Yeah I think I am still used to `for` loops :smile: